### PR TITLE
fix: Compass/waypoint now properly follows tracked content

### PR DESCRIPTION
**Pull request by magicus**

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/combat/ContentTrackerFeature.java [hunk 1]
```diff
@@ -29,14 +31,28 @@ public class ContentTrackerFeature extends Feature {
...
         if (playSoundOnUpdate.get()) {
             McUtils.playSoundUI(TRACKER_UPDATE_SOUND);
         }
     }
+
+    @SubscribeEvent
+    public void onSetSpawn(SetSpawnEvent e) {
+        if (!autoTrackCoordinates.get()) return;
+
+        Models.Compass.setCompassToSpawnTracker();
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/combat/ContentTrackerFeature.java:56]: Why?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/combat/ContentTrackerFeature.java:56]: Okay, I kind of see now..

**Review by kristofbolyai — APPROVED** [review]

File: common/src/main/java/com/wynntils/features/combat/ContentTrackerFeature.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/features/combat/ContentTrackerFeature.java:56]: Why what?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/combat/ContentTrackerFeature.java:56]: I didn't understand what the spawn point does in related to content tracking, but I checked now..

Comment (magicus) [reply common/src/main/java/com/wynntils/features/combat/ContentTrackerFeature.java:56]: Ah, maybe I should have commented on that.

**Pull request merged**